wordpress.org/public_html/wp-content/themes/pub/wporg-support-2024/inc/block-config.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Set up configuration for dynamic blocks.
+ */
+
```

Comment:
Should this file be namespaced? If not, maybe the function prefixes should be `wporg_support_`?

Reply:
This theme isn't generally using namespacing, but there are a few references around. I'll use it.